web-console/odc-about-developer-perspective-in-the-web-console.adoc

```diff
@@ -0,0 +1,32 @@
+ifdef::context[:parent-context-of-odc-about-developer-perspective-in-the-web-console: {context}]
+[id="odc-about-developer-perspective-in-the-web-console_{context}"]
+= About Developer perspective in the web console
+include::modules/common-attributes.adoc[]
+:context: odc-about-developer-perspective
+toc::[]
+
+The {product-title} web console provides two perspectives; the *Administrator* perspective and the *Developer* perspective.
+
+The *Developer* perspective provides the following developer focused features:
+
+* Ability to create and deploy applications on {product-title} by importing existing codebases, images, and dockerfiles.
+* Visual representation of all the applications within a project, their build status, and the components and services associated with the applications.
+* Ability to add required applications and services from the Developer catalog and group the components of an application.
+* Integration of the Serverless capabilities (Technical Preview).
```

Comment:
we say Technology Preview instead of Technical Preview per https://access.redhat.com/support/offerings/techpreview